easybuild/easyconfigs/v/vsc-mympirun/vsc-mympirun-4.0.0b0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'PythonPackage'
+
+name = 'vsc-mympirun'
+version = '4.0.0b0'
+
+homepage = 'https://github.com/hpcugent/vsc-mympirun'
+description = """VSC-tools is a set of Python libraries and scripts that are commonly used within HPC-UGent."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = [PYPI_SOURCE]
+sources = ['vsc-mympirun-%(version)s.tar.gz']
+
+dependencies = [
+    ('vsc-base', '2.5.8'),
+    ('IPy', '0.83'),
+]
+builddependencies = [('vsc-install', '0.10.25')]
+
+#postinstallcmds = ['echo "import pkg_resources; pkg_resources.declare_namespace(__name__)" > %(installdir)s/lib/vsc/__init__.py']
+#use_easy_install = True
+
+# we ship something in bin/fake
+modextrapaths = {'PATH': 'bin/fake'}
+
+#options = {'modulename': 'vsc.mympirun'}
+options = {'modulename': 'vsc'}
```

Comment:
Ah, yes, I need this hack because the `import` check fails if `vsc-base` is also installed system-wide.

Using `mympirun` itself works fine though, because that doesn't do a regular `import vsc.mympirun`, but uses a script controlled by `setuptools`.

I can include a `mympirun --help` sanity check command instead to verify that the installation is correct?